linode/manager
# feat: [M3-7163] - Add Firewall Section to NodeBalancer Details Page

## Description 📝
List Firewalls that are assigned to NodeBalancer in the `Summary` and the `Settings` tabs.

## Changes  🔄
List any change relevant to the reviewer.
- Display Firewall link in the NodeBalancer `Summary` tab
- Display Firewall info in the NodeBalancer `Settings` tab with the ability to unassign it.
- Display links to assign Firewalls and to Documentation.

## Preview 📷
| Before  | After   |
| ------- | ------- |
| ![dd](https://github.com/linode/manager/assets/119514965/2bef35fd-33ea-4a5a-83b6-28765af100c6) | ![cc](https://github.com/linode/manager/assets/119514965/24677f01-b87b-4268-9760-2f15ce979f2d) |
| ![bb](https://github.com/linode/manager/assets/119514965/ce9eeda7-1727-43b6-92e2-1af70db0f5f1) | ![aa](https://github.com/linode/manager/assets/119514965/cdb7319f-cd20-4f0a-8750-8173f4b5b7b7) |
| | ![ee](https://github.com/linode/manager/assets/119514965/10a15c71-0ebc-4e51-8a6e-7494d4d63cd4) |


## How to test 🧪

### Prerequisites
(How to setup test environment)
- Switch over to the `Dev` environment

### Verification steps 
(How to verify changes)
- Navigate to the NodeBalancers Settings page
- Verify that you see a Firewalls Accordion component
- If there are no Firewalls assigned, verify that  you can see a link to the Firewalls landing page and a separate link to the Firewalls documentation page.
- If there is a Firewalls attached, verify that you can see a Table with information pertaining to the Firewall.
- Verify that clicking the `Unassign` button removes the table row.
- Verify that after unassigning the Firewall, the two links are shown to the user again.

## As an Author I have considered 🤔

*Check all that apply*
